integration-tests/test/native-eth-ovm-calls.spec.ts

```diff
@@ -0,0 +1,309 @@
+import { BigNumber, Contract, ContractFactory, Wallet } from 'ethers'
+import { ethers } from 'hardhat'
+import chai, { expect } from 'chai'
+import { GWEI, fundUser, encodeSolidityRevertMessage } from './shared/utils'
+import { OptimismEnv } from './shared/env'
+import { solidity } from 'ethereum-waffle'
+import { sleep } from '../../packages/core-utils/dist'
+import {
+  getContractFactory,
+  getContractInterface,
+} from '../../packages/contracts/dist'
+
+chai.use(solidity)
+
+describe('Native ETH value integration tests', () => {
+  let env: OptimismEnv
+  let wallet: Wallet
+  let other: Wallet
+
+  before(async () => {
+    env = await OptimismEnv.new()
+    wallet = env.l2Wallet
+    other = Wallet.createRandom().connect(wallet.provider)
+  })
+
+  it('should allow an L2 EOA to send to a new account and back again', async () => {
+    const getBalances = async (): Promise<BigNumber[]> => {
+      return [
+        await wallet.provider.getBalance(wallet.address),
+        await wallet.provider.getBalance(other.address),
+      ]
+    }
+
+    const checkBalances = async (
+      expectedBalances: BigNumber[]
+    ): Promise<void> => {
+      const realBalances = await getBalances()
```

Comment:
This is a super bespoke function for just this `it` and not intended to be used otherwise.  I can change if you want, but I felt like this was sufficient.